packages/client/examples/node.js/storeCar.js

```diff
@@ -9,15 +10,17 @@ async function main() {
   const storage = new NFTStorage({ endpoint, token })
 
   // locally chunk'n'hash the file to get the CID and pack the blocks in to a CAR
-  const { root, car } = await packToBlob({
+  const { root, out } = await pack({
     input: fs.createReadStream('pinpie.jpg'),
   })
-
   const expectedCid = root.toString()
   console.log({ expectedCid })
 
-  // send the CAR to nft.storage, setting isCar to true
-  const cid = await storage.storeCar(car)
+  // Create the car reader
+  const carReader = await CarIndexedReader.fromIterable(out)
```

Comment:
No `fromIterable` on `CarIndexedReader` - you need to use `fromFile`.

Reply:
yes, I tested my local storeLargeCar instead of this file when pushed 🤦🏼‍♂️